rules/opentitan_test.bzl

```diff
@@ -152,7 +152,7 @@ def verilator_params(
         "@//hw:verilator",
         "@//hw:fusesoc_ignore",
     ]
-    required_tags = ["verilator"]
+    required_tags = ["verilator", "cpu:4"]
```

Comment:
I'd love to make this dynamic based on hw:verilator_options=--threads<x> or similar but I haven't figured out how to do it. 

This has been omitted in a number of tests that break when run in batch mode if resource based scheduling isn't overridden by a --local_test_jobs argument. The same argument offers the capability to schedule homogenous tests in CI as before.

If I build verilator for more threads I find it useful (if not intuitive) to change the number of CPUs the scheduler seeks to allocate: `build:fast --jobs=HOST_CPUS*2` which behaves reasonably well with a heterogeneous workload.